CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ This section is for maintaining a changelog for all breaking changes for the cli
 
 ### Fixed
 - Fix the deserialization of SortOptions ([#981](https://github.com/opensearch-project/opensearch-java/pull/981))
+- [BUG] Generic HTTP Actions in Java Client does not work with AwsSdk2Transport ([#978](https://github.com/opensearch-project/opensearch-java/pull/978))
```

Comment:
This is really a feature I think. I would say something like "Add support for AwsSdk2Transport to generic client" and move it to features.

Reply:
Yes and no, I consider it as a bug to be fair because the Generic HTTP Actions implementation completely missed this transport (but covered RestClient and AHC5). This is a core transport that we committed to support.

Reply:
IMO it's not something that we implemented but broke, it's something that was just not implemented. If you feel strongly about this, let's remove the word "[BUG]", it's already in the fixed section.